ocean_lib/models/data_token.py

```diff
@@ -295,7 +297,9 @@ def verify_transfer_tx(self, tx_id, sender, receiver):
         if tx_receipt.status == 0:
             raise AssertionError("Transfer transaction failed.")
 
-        logs = getattr(self.events, "Transfer")().processReceipt(tx_receipt)
+        logs = getattr(self.events, "Transfer")().processReceipt(
```

Comment:
Since we're touching this code anyway, lets upgrade it to use the ContractEvents API instead of `getattr`.  This is better because it helps IDEs navigate the codebase quicker. Like so:

```python
logs = self.events.Transfer().processReceipt(tx_receipt, errors=DISCARD)
```